vortex-duckdb-ext/src/scan.rs

```diff
@@ -9,14 +9,33 @@ use crate::duckdb::{
 };
 use crate::exporter::ArrayIteratorExporter;
 
+#[derive(Clone)]
 pub struct VortexBindData {
-    _first_file: VortexFile,
-    file_list: SegQueue<PathBuf>,
-    _column_names: Vec<String>,
+    first_file: VortexFile,
+    file_paths: Vec<PathBuf>,
+    column_names: Vec<String>,
     _column_types: Vec<LogicalType>,
 }
 
-pub struct VortexGlobalData {}
+impl std::fmt::Debug for VortexBindData {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
why do you impl this for only a few fields?

Reply:
pr is still in draft, but i wanted to get the unit tests working which require the asserted types to impl Debug.

Reply:
done for all relevant fields by now